### PR TITLE
mysql - readme fix

### PR DESCRIPTION
**Pull request by EugeneKorolevich**

### CODE REVIEW

File: 5/README.md [hunk 1]
```diff
@@ -3,12 +3,12 @@
 This image contains an installation MySQL 5.x.
 
 For more information, see the
-[Official Image Launcher Page](https://console.cloud.google.com/launcher/details/google/mysql5).
+[Official Image marketplace Page](https://console.cloud.google.com/marketplace/details/google/mysql5).
```
Comment (ovk6) [inline 5/README.md:6]: marketplace -> Marketplace

File: 5/README.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ Dockerfile for this image can be found [here](https://github.com/GoogleCloudPlat
 
 # <a name="using-kubernetes"></a>Using Kubernetes
 
-Consult [Launcher container documentation](https://cloud.google.com/launcher/docs/launcher-container)
+Consult [marketplace container documentation](https://cloud.google.com/marketplace/docs/container-images)
```
Comment (ovk6) [inline 5/README.md:50]: Marketplace

File: 5/README.md [hunk 3]
```diff
@@ -341,7 +341,7 @@ If your container was not started with a `MYSQL_ROOT_PASSWORD` value, substitute
 
 # <a name="using-docker"></a>Using Docker
 
-Consult [Launcher container documentation](https://cloud.google.com/launcher/docs/launcher-container)
+Consult [marketplace container documentation](https://cloud.google.com/marketplace/docs/marketplace-container)
```
Comment (ovk6) [inline 5/README.md:344]: Marketplace

File: 8/README.md [hunk 4]
```diff
@@ -47,7 +47,7 @@ Dockerfile for this image can be found [here](https://github.com/GoogleCloudPlat
 
 # <a name="using-kubernetes"></a>Using Kubernetes
 
-Consult [Launcher container documentation](https://cloud.google.com/launcher/docs/launcher-container)
+Consult [marketplace container documentation](https://cloud.google.com/marketplace/docs/container-images)
```
Comment (ovk6) [inline 8/README.md:50]: Marketplace

File: 8/README.md [hunk 5]
```diff
@@ -341,7 +341,7 @@ If your container was not started with a `MYSQL_ROOT_PASSWORD` value, substitute
 
 # <a name="using-docker"></a>Using Docker
 
-Consult [Launcher container documentation](https://cloud.google.com/launcher/docs/launcher-container)
+Consult [marketplace container documentation](https://cloud.google.com/marketplace/docs/marketplace-container)
```
Comment (ovk6) [inline 8/README.md:344]: Marketplace

**Review by ovk6 — CHANGES_REQUESTED** [review]
Move README to root directory (but preserving root README contents)

File: 5/README.md (hunk 1 above)
Comment (EugeneKorolevich) [reply 5/README.md:6]: done

File: 5/README.md (hunk 3 above)
Comment (EugeneKorolevich) [reply 5/README.md:344]: done

File: 8/README.md (hunk 5 above)
Comment (EugeneKorolevich) [reply 8/README.md:344]: done

**Review by ovk6 — APPROVED** [review]

**Pull request merged**